lib/util.js

```diff
@@ -76,14 +76,29 @@ exports.generateStreamArrayForLevels = function (lvls, stream) {
  * @return {Array}
  */
 exports.getDefaultStreams = function () {
-  return (env.get('TKE_LOG_SILENT') === 'true') ? [] : [{
-    stream: process.stdout,
-    level: exports.getLogLevel()
-  }, {
-    stream: process.stderr,
-    level: 'warn'
-  }, {
-    level: 'warn',
-    stream: require('./mongo-stream')
-  }];
+  if (env.get('TKE_LOG_SILENT') !== 'true') {
+    if (env.get('FH_MONGODB_CONN_URL')) {
+      return [{
+        stream: process.stdout,
+        level: exports.getLogLevel()
+      }, {
+        stream: process.stderr,
+        level: 'warn'
+      }, {
+        level: 'warn',
+        stream: require('./mongo-stream')
+      }];
+    } else {
```

Comment:
Instead of doing this, it's much easier to just start with an array of default streams (which are the ones writing to stdout/stderr), and if the the mongo env variable exists, then push the mongo stream onto that array as well. Less duplication of code.

Also, to reduce nesting, maybe just return `[]` immediately if TKE_LOG_SILENT = 'true'. Then you also won't have the `else return []` so far from the initial condition.